core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -371,4 +391,36 @@ export class NavigationComponent implements OnInit {
       });
   }
 
+  handleDisableOperatorIconChange() {
+    Observable.merge(
+      this.workflowActionService.getJointGraphWrapper().getJointOperatorHighlightStream(),
+      this.workflowActionService.getJointGraphWrapper().getJointOperatorUnhighlightStream(),
+      this.workflowActionService.getJointGraphWrapper().getJointGroupHighlightStream(),
+      this.workflowActionService.getJointGraphWrapper().getJointGroupUnhighlightStream(),
+      this.workflowActionService.getTexeraGraph().getDisabledOperatorsChangedStream(),
```

Comment:
as this block of code is showing many places, how about we do a merged observable inside WorkflowActionService to reduce duplication of code?

Reply:
this place is different than other places

the reason that you might feel it's a lot is because of there are so many highlight events, we can refactor them in to one single observable later